docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -527,6 +529,25 @@ The following things are validated:
 
 Any errors found will be logged with error level to ``stderr``.
 
+scrub
+^^^^^
+
+Rewrites the sstable, skipping or fixing corrupt parts. Not all kind of corruptions can be skipped or fixed by scrub.
+Currently this is limited to ordering issues on the partition, row or mutation-fragment level, see `sstable content <scylla-sstable-sstable-content_>`_ for more details.
+
+Scrub has several modes:
+
+* **abort** - abort the scrub as soon as any error is found (recognized or not). This mode is not all that useful (**validate** mode is strictly an improvement over **abort** mode), it is included for completeness' sake.
```

Comment:
```suggestion
* **abort** - Aborts the scrub as soon as any error is found (recognized or not). This mode is only included for the sake of completeness. We recommend using the **validate** mode so that all errors are reported.
```

Reply:
@denesb I'm not sure if this is what we want to say. You can rephrase it differently - I just think we shouldn't document anything as "not useful".